src/Components/Aspire.Azure.AI.Inference/ChatCompletionsClientSettings.cs

```diff
@@ -103,17 +103,17 @@ void IConnectionStringSettings.ParseConnectionString(string? connectionString)
             ConnectionString = connectionString
         };
 
-        if (connectionBuilder.TryGetValue(nameof(DeploymentId), out var modelId))
+        if (connectionBuilder.TryGetValue("DeploymentId", out var modelId))
```

Comment:
Why this change?

Reply:
I don't like to mix property names and string when they don't really technically are the same thing. In case we decide to change the property name this would not work anymore. Or the generation of the connection string should also use the `nameof` of the property. Can go back to nameof if you prefer it.